README.md

```diff
@@ -16,7 +16,7 @@ gem 'jekyll-github-metadata'
 Then go ahead and run `bundle install`. Once you've done that, add your repo & the gem to your `_config.yml`:
 
 ```yaml
-# or use PAGES_REPO_NWO in the env
+# or use PAGES_REPO_NWO in the env <-- what does this mean?
```

Comment:
this means that you set the environment variable `PAGES_REPO_NWO` to your repo as "username/repo" when you run Jekyll. In UNIX, you can do:

```
PAGES_REPO_NWO=username/repo bundle exec jekyll serve
```